README.md

```diff
@@ -11,14 +11,11 @@ END_METADATA -->
 
 # Webhooks API
 
-💥 Work in progress 💥
-
-You can **subscribe** to published **event types** by registering webhooks and
-will then be **notified** when any matching events are published that match the
+You can subscribe to published *event types* by registering webhooks.
+You will then be notified when any matching events are published for the
 given sales unit ID.
 
 As an example, a merchant can register a callback URL to receive a callback when
-any of the payments of the merchant is captured.
+any of the merchant's payments are captured.
 
-There is a set limit on how many registrations per event types per sales unit
-allowed, see the API reference for the exact limit.
+You can register for 5 webhooks per event type for each sales unit.
```

Comment:
I think we'd prefer to keep this info in one place only, would it be okay if you want to add it here that I removed it from the API, or vice versa?